src/views/Login.vue

```diff
@@ -8,13 +8,14 @@
     <div class="login-container">
       <div class="card">
         <img :src="$config.loginLogoUrl" class="pt-2 pb-5" />
-        <form @submit.prevent="submit">
+        <form @submit.prevent="submit" autocomplete="off">
           <div class="field mb-5">
             <p class="control has-icons-left has-icons-right">
               <input
                 v-model.trim="email"
                 class="input"
                 placeholder="Courriel"
+                autocomplete="new-password"
```

Comment:
Why turn off autocomplete ?
That's a useful feature. If the email is filled in, and the user's browser fills in the password automatically, then good for the user because he certainly opted in for the browser to remember the password in the first place.

Reply:
yes, i put it there for testing peruse, thanks